### ISSUE TITLE
About training

### ISSUE THREAD

**Issue by zhenhao-li** [Easquel/WaveDM#10]
Dear author,
﻿
I have recently read your paper: " WaveDM: Wavelet-Based Diffusion Models for Image Restoration ". The work was so solid and creative and it was attractive to me. And thank you for being willing to share your code, but there is a question I want to consult about.
﻿
First of all, I have noticed that you employ a combined objective function to optimize the network. The L1 loss function was used for optimizing HFRM while the MSE loss or the  L_simple for the diffusion process. But in the codebase you provided have the training_hrfm.py file. And in the training code, it seem like the HFRM was load from a checkpoint weight without participating in training. ( in the /WaveDM/models/ddm_wavelet.py line143 and line147 ) So the HFRM and DPM were training splitly or just I missing somthing else?
﻿
Thank you very much for your patience in reading and I am looking forward to your early reply.

**Comment by Easquel** [comment]
Actually, the two training options achieve similar results in the end. For the convenience of implementation, we split the training here.
